CS342MemberViews/Sources/CaitlinKunchur/ListensRow.swift

```diff
@@ -0,0 +1,93 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import InstructorViews
+import SwiftUI
+
+public struct ListensRow: View {
+        public var body: some View {
+            ZStack {
+            Color(hue: 0.5639, saturation: 0.0, brightness: 1)
+            
+            VStack(alignment: .leading) {
+                Text("Recent Listens").font(.custom(
+                    "GillSans",
+                    fixedSize: 25))
+                    .padding(.leading, 15)
+                    .padding(.top, 5)
+                    .foregroundStyle(.blue.gradient)
+                
+                ScrollView(.horizontal, showsIndicators: false) {
+                    HStack {
+                        VStack {
+                            Bundle.module.image(fromFileNamed: "car-cover")
```

Comment:
Similar to the `HobbiesRow.swift`, I would suggest moving the views into smaller chunks or using a `ForEach` loop to create your views.